scripts/build/gn_gen_cirque.sh

```diff
@@ -27,10 +27,16 @@ set -x
 
 env
 
-echo "Build: GN configure"
+cd "$ROOT_PATH"
 
-gn --root="$CHIP_ROOT" gen --check --fail-on-unused-args "$CHIP_ROOT/out/debug" --args='target_os="all"'"chip_build_tests=false chip_enable_wifi=false chip_im_force_fabric_quota_check=true enable_default_builds=false enable_host_gcc_build=true enable_standalone_chip_tool_build=true enable_linux_all_clusters_app_build=true enable_linux_lighting_app_build=true"
+echo "Ensure submodules for Linux builds are checked out"
+.scripts/checkout_submodules.py --shallow --platform linux
 
-echo "Build: Ninja build"
+echo "Setup build environment"
+source "scripts/bootstrap.sh"
```

Comment:
Won't this pull all the submodules?

Reply:
It seems to do so, but I did not expect it. In our yaml files we do the checkout_submodules then bootstrap and it is only a subset. I will need to understand this...

Reply:
switched bootstrap with activate. Will check cirque job logs to see if that worked.